packages/kolibri-common/components/SearchFiltersPanel/AccordionSelectGroup.vue

```diff
@@ -412,31 +408,32 @@
     margin-bottom: 1em;
   }
 
+  .category-button {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    padding: 0.25em 0.5em;
+    font-weight: normal;
+    text-align: left;
+    text-transform: none;
+  }
+
   .category-icon {
-    position: absolute;
-    top: 50%;
-    left: 0.5em;
-    width: 32px;
-    height: 32px;
-    transform: translateY(-50%);
+    order: 1;
```

Comment:
I dont think this order prop is doing anything, since the flex items are already in order :sweat_smile: